ibm/service/cis/resource_ibm_cis_alert_webhook_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package cis_test
+
+import (
+	"fmt"
+	"testing"
+
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMCisWebhooks_Basic(t *testing.T) {
+
+	webhookname := "test-Webhooks"
+	webhooksurl := "https://hooks.slack.com/services/Ds3fdBFbV/1234568"
+	webhooksecret := "ff1d9b80-b51d-4a06-bf67-6752fae1eb74"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheckCis(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCisWebhooks_basic("test", acc.CisDomainStatic),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_cis_webhook.test", "name", webhookname),
+					resource.TestCheckResourceAttr("ibm_cis_webhook.test", "url", webhooksurl),
+					resource.TestCheckResourceAttr("ibm_cis_webhook.test", "secret", webhooksecret),
+				),
+			},
+		},
+	})
+}
+func testAccCheckCisWebhooks_basic(id, CisDomainStatic string) string {
+	return testAccCheckIBMCisDomainDataSourceConfigBasic1() + fmt.Sprintf(`
+	resource "ibm_cis_webhook"  "%[1]s"  {
+		cis_id 		= "crn:v1:staging:public:internet-svcs-ci:global:a/01652b251c3ae2787110a995d8db0135:79c0ce9a-f0fd-4c10-ae78-d890aca7a350::"
```

Comment:
Donot hardcode any crn
Please update test results to this PR by making changes to this template..(fetch id from datasource)

Reply:
Changed